packages/portfolio-contract/test/snapshots/portfolio.contract.test.ts.md

```diff
@@ -78,6 +114,94 @@ Generated by [AVA](https://avajs.dev).
       },
     }
 
+> Under "published", the "pendingTxs" node is delegated to ymax.
+> The example below illustrates the schema of the data published there.
+> 
+> See also board marshalling conventions (_to appear_).
+
+    [
+      [
+        'published.pendingTxs.tx0',
+        {
+          dst: [
```

Comment:
`dst` seems inconsistent with the precedent of `dest` elsewhere in the contract. hm.

Reply:
I've renamed `dst` to `dest`.